eng/format.ps1

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+dotnet tool run dotnet-format $Args
```

Comment:
❓ Why is this file part of this pull request?
📝 File is missing the final newline

Reply:
If a final newline is required, then we should update the .editorconfig with `insert_final_newline = true` for .ps1 and .sh files.